invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/RDMDepositForm/RDMDepositForm.js

```diff
@@ -113,6 +114,49 @@ export class RDMDepositForm extends Component {
             { text: "Other", value: "other" },
           ],
         },
+
+        // TODO: Replace with an API backend
+        funding: {
```

Comment:
moderate: This does not match the inner data model (maybe intended?). Here there is a `funding` dictionary with a list of `funder` and a list of `award`, while in the data model there is a list of `funding` (and each funding can have one `funder and/or one `award`).

Reply:
Yes this is intended. This is the (placeholder) vocabulary, not the actually stored values in the field. When the field is filled we store an array of funding objects:` [{ funder: {...}, award: {...} }, ... ]`.

I was wondering about the schema in invenio-rdm-records, so it's good you brought it up. There can be just a funder? or just an award? The frontend assumes that an award necessarily has a funder. We can go over this on Monday.

Reply:
It's important to be able to support a Funder without an Award. A lot of times we don't have information about what specific award is associated with a work, or the funding is from an organization that doesn't use awards/grants. 

Having an award require a funder seems like a reasonable requirement, but I'm not sure whether that will cause troubles for Zenodo (I think they accept only award numbers, then map separately)

Reply:
Good to know!

Reply:
Thanks @tmorrell! So If I understand correctly the `funder` is always required but the `award` might or might not be present (@fenekku we need to crosscheck this since it's not consistent with the data model atm). In addition, @fenekku since the front end separates them, how will is associate one funder to award? Assuming positions (i.e. [0] and [0] etc. if so need to add an empty award when there's not so keep the index consistent).

Reply:
After discussion, this is just a placeholder. It will be revisited in next Invenio Sprint: https://github.com/inveniosoftware/invenio-rdm-records/issues/264 . :+1: